Gemfile

```diff
@@ -64,3 +63,7 @@ group :test do
   gem 'database_cleaner'
   gem 'pry-byebug'
 end
+
+group :test, :production do
+  gem 'mysql2'
```

Comment:
Shouldn't sqlite3 be used for 'test'? That is for rspec, not for the test environment.